index.js

```diff
@@ -496,3 +496,34 @@ function getWildcardNode (node, method, path, len) {
   }
   return null
 }
+
+/**
+ * Returns the position of the last closing parenthese of the regexp expression.
+ *
+ * @param {String} path
+ * @param {Number} idx
+ * @returns {Number}
+ * @throws {TypeError} when a closing parenthese is missing
+ * @private
+ */
+function getClosingParenthensePosition (path, idx) {
+  // `path.indexOf()` will always return the first position of the closing parenthese,
+  // but it's inefficient for grouped or wrong regexp expressions.
+  // see issues #62 and #63 for more info
+
+  var parentheses = 1
+
+  while (idx < path.length) {
+    idx++
+
```

Comment:
if the next char is `\` then you need to ignore the next one for it to be spec compliant :)

```
if (path[idx] === '\\') {
  continue
}
```

Reply:
Hi @mafintosh can give a use case using backslash. thanks

Reply:
Sure, `find.get(/foo/(bar/\(baz\)))`, matches '/foo/bar(baz)'.